continuous_integration/scripts/build-docs

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
Rather than having an entirely new script, can we update [build-legate](https://github.com/nv-legate/legate.core/blob/branch-24.01/continuous_integration/scripts/build-legate#L62) to have a docs type?